### ISSUE TITLE
 How to set  parameters when training mobilenet on imagenet1k?

### ISSUE THREAD

**Issue by BoXiao123** [apache/incubator-mxnet#8441]
I'm trying to train the mobilenet on the imagenet1k by using the train_imagenet script. However, the train accuracy has achieved 0.30 after 5 epoches and it didn't go up any-more. Here is my parameter used for training mobilenet:
    
INFO:root:start with arguments Namespace(batch_size=128, benchmark=0, data_nthreads=4, data_train='/home/b_xi/disk_files_824G/train/imagenet1k.rec', data_train_idx='/home/b_xi/disk_files_824G/train/imagenet1k.idx', data_val=None, data_val_idx='', disp_batches=20, dtype='float32', gpus='0,1', image_shape='3,224,224', kv_store='device', load_epoch=20, lr=0.08, lr_factor=0.1, lr_step_epochs='30,60', max_random_aspect_ratio=0.25, max_random_h=36, max_random_l=50, max_random_rotate_angle=10, max_random_s=50, max_random_scale=1, max_random_shear_ratio=0.1, min_random_scale=1, model_prefix='/home/b_xi/disk_files_824G/mxnet_api/train', mom=0.9, monitor=0, network='mobilenet', num_classes=1000, num_epochs=60, num_examples=1281167, num_layers=14, optimizer='sgd', pad_size=0, random_crop=1, random_mirror=1, rgb_mean='123.68,116.779,103.939', test_io=0, top_k=0, wd=0.0001)

## What have you tried to solve it?
have someone trained mobilenet on imagenet1k could tell me how to set these parameters in order to 
get good results.

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by b0noI** [comment]
Suggested tags: "Question", "HowTo"

**Comment by zhreshold** [comment]
You can checkout this https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html, and for this kind of questions I suggest to use [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)).
Closing this for now, feel free to contact me to reopen it.
